### ISSUE TITLE
northrend instance glitched

### ISSUE THREAD

**Issue by Imisstheoldkanye** [dalaranwow/dalaran-wow#5592]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I had to log off while doing the quest in icecrown where you play as arthas (same mechanics as a vehicle), killing your soldiers and raising them from the dead. when i logged back in, my character was no longer arthas, but the instance of northrend was the same. this means that there are no NPCs or interactable objects loaded, i cannot see or interact with other players in northrend, and dungeons are inactive (I can enter violet hold, but it's just an empty room. I tried to abandon the quest to exit the instance, however this did not work. the zepplins still run though, so i can to vanilla and bc without any problems. the quest giver is not in the instance, so i cannot re-accept and complete the quest.
**Expected behaviour**: Tell us what should happen instead
when i abandon the quest it should've taken me out of the instance.
**Steps to reproduce the problem**:

1.  accept the quest
2.  close the game (alt+f4; any method outside of the game)
3. re-enter the game

please get me out of it, this is spaceforce, level 78 warrior, on algalon

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![northrendinstanceglitch](https://user-images.githubusercontent.com/32621283/44609222-6aa9c000-a7bc-11e8-8173-36a0db31ac0f.PNG)

**Comment by Golganeth** [comment]
It should be fine already, if the issue persists:

1) Clear your Cache folder
2) Kill your character on purpose
3) Abandon, re-take or complete any quest that could be messing with your phasing

Should the issue persist or happen to you again, and the steps above didn't work, please don't hesitate to take this to the in-game support ticket system and we will help you out.

Closing up.
